test/server/liveReload-option.test.js

```diff
@@ -1,124 +0,0 @@
-'use strict';
```

Comment:
We don't need this stupid test anymore, we have good e2e tests right now https://github.com/webpack/webpack-dev-server/blob/master/test/e2e/hot-and-live-reload.test.js